controllers/contacts.js

```diff
@@ -0,0 +1,66 @@
+const Joi = require("joi");
+
+const { HttpError, ctrlWrapper } = require("../helpers");
+
+const contacts = require("../models/contacts");
+
+const addShema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
You should validate phone and email properly. Schema should be in separated file